client/src/lib/attempt-result.js

```diff
@@ -364,9 +364,9 @@ export function attemptResultsWarning(
   const skippedGapIndex =
     trimTrailingSkipped(attemptResults).indexOf(SKIPPED_VALUE);
   if (skippedGapIndex !== -1) {
-    return `You've omitted attempt ${
+    return [`You've omitted attempt ${
```

Comment:
Please change to an object as suggested above. Note that we also need to update tests in `attempt-result.test.js` :)